documentation/docs/requirements/system-overview.md

```diff
@@ -6,14 +6,15 @@ sidebar_position: 1
 
 ## Project Abstract
 
-Devs and Dragons is a multiplayer web-based application that provides a gamified learning platform for computer science students and educators. Users must form parties of two to four members to take on quests that involve solving coding problems. Alternatively, users can create guilds to hold multiple parties and assign multiple quests. Each user in a party takes turns coding to solve the problem with a given time constraint based on difficulty. Each quest and the problems created for the quest are generated using user input and a GPT bot. 
+Devs and Dragons is a multiplayer web-based application designed as a gamified learning platform for computer science students and educators. Users can form parties of two to four members to tackle quests involving coding challenges. Team members take turns solving problems within a time limit determined by the selected difficulty level. Quests and their coding problems are generated by a GPT bot based on user inputs, including problem type, difficulty, and theme description.
```

Comment:
- it is no longer for educators
- theme description -> quest description
- also mention that we are grading user answers with GPT